azext_iot/iothub/commands_state.py

```diff
@@ -15,10 +15,10 @@ def state_export(
     resource_group_name: Optional[str] = None,
     login: Optional[str] = None,
     auth_type_dataplane: Optional[str] = None,
-    overwrite_file: Optional[bool] = False
+    force: Optional[bool] = False
```

Comment:
The use of `Optional[x]` typing is when `None` is an acceptable value. But in this case `force` is either `True` or `False`, right? If so, you can remove `Optional[]` and just keep the `bool` typing.